apis/kubedb/v1/helpers.go

```diff
@@ -159,3 +160,10 @@ func UsesAcmeIssuer(kc client.Client, ns string, issuerRef core.TypedLocalObject
 		return false, fmt.Errorf("invalid issuer %+v", issuerRef)
 	}
 }
+
+func GetNetworkPolicyLabels() map[string]string {
+	return map[string]string{
+		meta_util.ComponentLabelKey: kubedb.ComponentDatabase,
```

Comment:
Should we update this label? It's not necessarily a database component.

Reply:
Changed the func name to `GetSelectorForNetworkPolicy()`